etl_framework/config_mixins/FiltersMixin.py

```diff
@@ -29,6 +30,26 @@ def add_filters(self, filter_mappings):
             self.set_filter(getattr(filter_mappings, self.get_filter()))
             self.set_post_filter(getattr(filter_mappings, self.get_post_filter()))
 
+    def add_filters_from_module(self, filter_module):
```

Comment:
yeah i just did it this way because i didn't want to break any existing stuff relying on the else part